test/JDBC/expected/TestSpatialFun-vu-verify.out

```diff
@@ -0,0 +1,932 @@
+~~END~~
+
+
+DECLARE @isEqual BIT = 1;
+SELECT ID, PointColumn1.STEquals(PointColumn2) AS isEqual,
+CASE WHEN PointColumn1.STEquals(PointColumn2) = @isEqual THEN 'yes' ELSE 'no'
+END AS isEqual
+FROM YourTableTemp2;
+go
+~~START~~
+int#!#bit#!#text
+1#!#0#!#no
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+WITH EqualCTE AS ( SELECT ID, PointColumn.STEquals(@referencePoint) AS Equality FROM YourTableTemp)
+SELECT * FROM EqualCTE WHERE Equality <= 1;
+go
+~~START~~
+int#!#bit
+1#!#0
+2#!#0
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326); 
+UPDATE YourTableTemp SET PointColumn = @referencePoint
+WHERE PointColumn.STEquals(@referencePoint) != 1;
+go
+~~ROW COUNT: 2~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326); 
+UPDATE YourTableTemp SET PointColumn = @referencePoint
+WHERE @referencePoint.STEquals(PointColumn) = 1;
+go
+~~ROW COUNT: 2~~
+
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+WITH EqualCTE AS ( SELECT ID, PointColumn.STEquals(@referencePoint) AS Equal FROM YourTableTemp ORDER BY PointColumn.STX)
+SELECT * FROM EqualCTE WHERE Equal <= 1.0 ORDER BY Equal;
+GO
+~~START~~
+int#!#bit
+1#!#1
+2#!#1
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+WITH EqualCTE AS ( SELECT ID, @referencePoint.STEquals(PointColumn) AS Equal FROM YourTableTemp ORDER BY PointColumn.STX)
+SELECT * FROM EqualCTE WHERE Equal <= 1.0 ORDER BY Equal;
+GO
+~~START~~
+int#!#bit
+1#!#1
+2#!#1
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+DECLARE @isEqual BIT = 1;
+SELECT ID, PointColumn.STEquals(@referencePoint) AS EqualityReferencePoint,
+CASE WHEN PointColumn.STEquals(@referencePoint) <= @isEqual THEN 'Close'
+ELSE 'Far'
+END AS Proximity
+FROM YourTableTemp;
+go
+~~START~~
+int#!#bit#!#text
+1#!#1#!#Close
+2#!#1#!#Close
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+SELECT ID, @referencePoint.STEquals(PointColumn) AS EqualityReferencePoint,
+CASE WHEN @referencePoint.STEquals(PointColumn) <= @referencePoint.STY THEN 'Close'
+ELSE 'Far'
+END AS Proximity
+FROM YourTableTemp;
+go
+~~START~~
+int#!#bit#!#text
+1#!#1#!#Far
+2#!#1#!#Far
+~~END~~
+
+
+DECLARE @Ranges TABLE (MinDistance float, MaxDistance float);
+INSERT INTO @Ranges VALUES (0, 5), (5, 10), (10, 15);
+SELECT * FROM ( SELECT ID,
+CASE WHEN PointColumn1.STEquals(PointColumn2) BETWEEN 0 AND 1 THEN 'yes'
+ELSE 'no'
+END AS Range
+FROM YourTableTemp2
+) AS Source
+PIVOT ( COUNT(ID) FOR Range IN ([0-5], [5.1-10], [10.1-15], [15.1+])) AS PivotTable;
+go
+~~ROW COUNT: 3~~
+
+~~START~~
+int#!#int#!#int#!#int
+0#!#0#!#0#!#0
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+SELECT ID, PointColumn.STEquals(@referencePoint) AS Equality FROM <YourDatabase>.dbo.YourTableTemp;
```

Comment:
What is the purpose of this test? `<YourDatabase>` has special characters so it will always through error anyways.
If the motive is to test three part names then use a valid identifier for database name.

Reply:
resolved